tests/functional/lib/bash/test_header

```diff
@@ -949,5 +960,11 @@ _get_test_host
 # provides: CYLC_TEST_INSTALL_TARGET
 _get_test_install_target
 
+if ${CYLC_SHOW_TEST_EXECUTION:-false}; then
+    # write the name of tests as they are run
+    # (helpful for debugging hanging tests)
+    echo "# $TEST_NAME_BASE" >&2
+fi
```

Comment:
Minor feature to help with diagnosing hanging tests, this writes the name of each test just before it starts running.